app/views/support_interface/performance/index.html.erb

```diff
@@ -50,7 +50,7 @@
 </p>
 
 <p class='govuk-body'>
-  <%= govuk_link_to 'Download referee survey results (CSV)', support_interface_referee_survey_path %>
+  <%= govuk_link_to 'Download referee survey results (CSV)', support_interface_referee_survey_path, download: 'apply-referee-survey-results.csv' %>
```

Comment:
The provider user and referee/candidate survey controller actions create a CSV as an attachment. I'm not 100% sure how this interacts with the HTML download attribute.

Reply:
👍 I didn't check, will remove